packaging/ubuntu-16.04/snapd.install

```diff
@@ -18,6 +18,8 @@ data/udev/rules.d/66-snapd-autoimport.rules /lib/udev/rules.d
 data/info /usr/lib/snapd/
 # polkit actions
 data/polkit/io.snapcraft.snapd.policy /usr/share/polkit-1/actions/
+# apt hook
+data/apt/20snapd.conf /etc/apt/apt.conf.d/
```

Comment:
I wish apt would pick hooks from /lib as this is yet another thing that needs to be in /etc

Reply:
Yeah :/

Reply:
Since this is a new feature, can we influence apt to do the reasonable thing?